pkg/envtest/webhook.go

```diff
@@ -40,8 +40,8 @@ import (
 
 // WebhookInstallOptions are the options for installing mutating or validating webhooks
 type WebhookInstallOptions struct {
-	// Paths is a list of paths to the directories containing the mutating or validating webhooks yaml or json configs.
-	DirectoryPaths []string
+	// Paths is a list of paths to the directories or files containing the mutating or validating webhooks yaml or json configs.
+	Paths []string
```

Comment:
Renaming `DirectoryPaths ` to `Paths` would be a breaking change right?
If so, we should probably mark this commit with a :warning: prefix.

Unless we want to avoid that and add a separate field e.g `FilePaths`.

Reply:
yeah, you are right, it is a breaking change.

I prefer to not add `FilePaths` since I would like to keep it consistent with CRDInstallOptions.

WDYT? @vincepri @DirectXMan12

Reply:
+1 for consistency, we can delay this change until v0.7